projects/1.16/assets/tinkers-archery/tinkersarchery/lang/zh_cn.json

```
@@ -0,0 +1,247 @@
+  "modifier.tinkersarchery.power.3": "强力Ⅲ",
+  "modifier.tinkersarchery.power.4": "强力Ⅳ",
+  "modifier.tinkersarchery.power.5": "强力Ⅴ",
+  "modifier.tinkersarchery.power.description": "箭头造成更多伤害",
+  "modifier.tinkersarchery.power.flavor": "怎么做到的?",
+  "modifier.tinkersarchery.slicing": "切割",
+  "modifier.tinkersarchery.slicing.description": "箭总是危险的",
+  "modifier.tinkersarchery.slicing.flavor": "削铁如泥",
+  "modifier.tinkersarchery.superslime": "超级史莱姆",
+  "modifier.tinkersarchery.superslime.description": "你弓上的黏液覆层越多，你拉弓就越快，箭也飞得越远",
+  "modifier.tinkersarchery.superslime.draw_speed": "超级史莱姆拉弓速度",
+  "modifier.tinkersarchery.superslime.elasticity": "超级史莱姆拉弓磅数",
+  "modifier.tinkersarchery.superslime.flavor": "Duang Duang的",
+  "modifier.tinkersarchery.swiftstrike": "移动射击",
+  "modifier.tinkersarchery.swiftstrike.attack_damage": "移动伤害",
+  "modifier.tinkersarchery.swiftstrike.description": "移动速度加快会造成更多伤害",
+  "modifier.tinkersarchery.swiftstrike.flavor": "放轻松~",
+  "modifier.tinkersarchery.twirling_trajectory": "转动轨迹",
+  "modifier.tinkersarchery.twirling_trajectory.description": "箭在空中旋转",
+  "modifier.tinkersarchery.twirling_trajectory.flavor": "做一个飞行特技表演",
+  "modifier.tinkersarchery.uplifting": "仰望",
+  "modifier.tinkersarchery.uplifting.description": "箭头稍微向上射出以帮助瞄准",
+  "modifier.tinkersarchery.uplifting.flavor": "举头望明月",
+  "modifier.tinkersarchery.weighty": "稳定",
+  "modifier.tinkersarchery.weighty.description": "使箭头更稳定，使其飞出更讨喜的轨迹",
+  "modifier.tinkersarchery.weighty.flavor": "芝麻稳定",
+  "pattern.tinkersarchery.arrow_fletching": "箭羽",
+  "pattern.tinkersarchery.arrow_shaft": "箭杆",
+  "pattern.tinkersarchery.arrowhead": "箭头",
+  "pattern.tinkersarchery.bowguide": "导向器",
+  "pattern.tinkersarchery.bowshaft": "弓臂",
+  "pattern.tinkersarchery.bowstring": "弓弦",
+  "pattern.tinkersarchery.crossbow_arm": "弩臂",
+  "pattern.tinkersarchery.large_bowshaft": "大型箭杆",
+  "stat.tinkersarchery.arrow_fletching": "箭羽",
+  "stat.tinkersarchery.arrow_shaft": "箭杆",
+  "stat.tinkersarchery.arrowhead": "箭头",
+  "stat.tinkersarchery.bow": "弓",
+  "stat.tinkersarchery.bowguide": "导向器",
+  "stat.tinkersarchery.bowstring": "弓弦",
+  "tool_stat.tinkersarchery.accuracy": "准确度: ",
+  "tool_stat.tinkersarchery.accuracy.description": "射击有多准确。\n这受弓和箭的共同影响。",
+  "tool_stat.tinkersarchery.accuracy.multiplier_description": "弓具的总精度将乘以此值。",
+  "tool_stat.tinkersarchery.count": "数量: ",
+  "tool_stat.tinkersarchery.count.description": "射出箭的数目。",
+  "tool_stat.tinkersarchery.count.multiplier_description": "射出箭的总数将与此相乘。",
+  "tool_stat.tinkersarchery.draw_speed": "拉弓速度: ",
+  "tool_stat.tinkersarchery.draw_speed.description": "拉弓速度有多快。\n箭的射出速度与拉弓程度不是线性关系。",
+  "tool_stat.tinkersarchery.draw_speed.multiplier_description": "弓具的拉弓速度将乘以此值。",
+  "tool_stat.tinkersarchery.elasticity": "磅数: ",
+  "tool_stat.tinkersarchery.elasticity.description": "弓具射出的箭有多快。\n箭的速度参与计算伤害。",
+  "tool_stat.tinkersarchery.elasticity.multiplier_description": "弓具的总磅数将乘以此值。",

```

```suggestion
  "tool_stat.tinkersarchery.elasticity": "拉力：",
  "tool_stat.tinkersarchery.elasticity.description": "弓弩射出的箭有多快。\n箭的速度参与伤害计算。",
  "tool_stat.tinkersarchery.elasticity.multiplier_description": "弓弩的总拉力将乘以此值。",
```
官库已全部修改为Elasticity（弹性）